sceptre/config/reader.py

```diff
@@ -475,7 +459,7 @@ def _render(self, directory_path, basename, stack_group_config):
             message = f"{Path(directory_path, basename).as_posix()} - {err}"
 
             if logging_level() == logging.DEBUG:
-                debug_file_path = self._write_debug_file(
+                debug_file_path = write_debug_file(
                     json.dumps(self.templating_vars), prefix="vars_"
```

Comment:
Hey, in all these places where we're dumping templating vars, we should dump with `indent=4`. That is going to make the debug files far more human-readable than a dense blob of unpretty json.

Reply:
Ok, done.